app/server.rb

```diff
@@ -665,6 +665,28 @@ def getChildDepts(unit)
 end
 
 ###################################################################################################
+# Item Metrics 
+get "/api/item/metrics" do |shortArk|
```

Comment:
@martinhaye  This Api call hasn't been rigged up yet.  But to help with performance we could avoid grabbing this metrics .....  usage data until the user clicks on the Metrics tab. What d'ya say?